#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_autoupdate.dpatch by Stéphane Marguet (Stemp) <stempubuntu@gmail.com>
##
## DP: This is a patch for turning off autoupdate. This is done to avoid the
## undesarible pop up when a new version of aMSN is avaible because we are the
## ones packing the application to our users. 

@DPATCH@
diff -urNad amsn-0.98b-svn11364.old~/config.tcl amsn-0.98b-svn11364.old/config.tcl
--- amsn-0.98b-svn11364.old/config.tcl	2009-07-06 19:34:30.000000000 +0200
+++ amsn-0.98b-svn11364/config.tcl	2009-07-22 14:15:19.000000000 +0200
@@ -233,7 +233,7 @@
 		::config::setKey autoconnect 0			;#Automatically connect when amsn starts
 		::config::setKey libtls_temp ""			;#TLS
 		::config::setKey lineflushlog 1			;#Flush log files after each line
-		::config::setKey autocheckver 1			;#Automatically check for newer versions on startup
+		::config::setKey autocheckver 0			;#Automatically check for newer versions on startup
 		::config::setKey truncatenames 1		;#Truncate nicknames longer than window width in windows' title
 		::config::setKey truncatenicks 0		;#Truncate nicknames longer than window width in chat windows
 		::config::setKey keepalive 1				;#Keep alive connection (ping every minute)
@@ -244,7 +244,7 @@
 		::config::setKey start_ns_server "messenger.hotmail.com:1863"
 		::config::setKey default_gateway_server "gateway.messenger.hotmail.com"
 		::config::setKey start_gateway_server "gateway.messenger.hotmail.com"		
-		::config::setKey activeautoupdate 1		;#Active the auto update
+		::config::setKey activeautoupdate 0		;#Active the auto update
 		::config::setKey allowbadwords 1		;#Disable censure on nicks
 		::config::setKey enablebanner 1		;#Show or Hide AMSN Banner (By default Show)
 		::config::setKey startontray 0		;#Start amsn on tray icon only (hide contact list)
#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_checkupdate.dpatch by Stéphane Marguet (Stemp) <stempubuntu@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove Check Update Option.

@DPATCH@
diff -urNad amsn-0.98b-svn11364.old~/login_screen.tcl amsn-0.98b-svn11364.old/login_screen.tcl
--- amsn-0.98b-svn11364.old~/login_screen.tcl	2008-06-26 22:17:53.000000000 +0200
+++ amsn-0.98b-svn11364.old/login_screen.tcl	2009-07-22 16:25:00.000000000 +0200
@@ -44,9 +44,6 @@
 	component service_status_link
 	component new_account_link
 
-	component check_ver_icon
-	component check_ver_text
-
 	variable more_label
 
 	variable remember_me 0
@@ -89,7 +86,6 @@
 		contentmanager add group login_screen main checkboxes auto_login -orient horizontal	-widget $self	-pady 2
 		contentmanager add group login_screen main login		-orient horizontal	-widget $self	-align center	-pady 8
 		contentmanager add group login_screen main links		-orient vertical	-pady 32	-widget $self	-align left
-		contentmanager add group login_screen main check_ver		-orient horizontal	-pady 8		-widget $self	-align center
 		contentmanager add group login_screen main more			-orient horizontal	-pady 8		-widget $self	-align right
 
 		# Create widgets
@@ -159,9 +155,6 @@
 		set service_status_link [$self create text 0 0 -anchor nw -text [trans msnstatus]]
 		# New account
 		set new_account_link [$self create text 0 0 -anchor nw -text [trans new_account]]
-		# Check for newer amsn version
-		set check_version_icon [$self create image 0 0 -anchor nw -image [::skin::loadPixmap download]]
-		set check_version_text [$self create text 0 0 -anchor nw -text [trans checkver] -fill [::skin::getKey loginurlfg]]
 
 		set more_label [$self create text 0 0 -anchor nw -text [trans more]]
 
@@ -194,8 +187,6 @@
 		contentmanager add element login_screen main links forgot_pass -widget $self -tag $forgot_pass_link -pady 2
 		contentmanager add element login_screen main links service_status -widget $self -tag $service_status_link -pady 2
 		contentmanager add element login_screen main links new_account -widget $self -tag $new_account_link -pady 2
-		contentmanager add element login_screen main check_ver icon -widget $self -tag $check_version_icon -padx 4 -valign middle
-		contentmanager add element login_screen main check_ver text -widget $self -tag $check_version_text -padx 4 -valign middle
 		contentmanager add element login_screen main more label	-widget $self	-tag $more_label
 
 		# Set font for canvas all text items
@@ -241,7 +232,6 @@
 		$self CanvasTextToLink login_screen main links forgot_pass [list launch_browser "https://accountservices.passport.net/uiresetpw.srf?lc=1033"]
 		$self CanvasTextToLink login_screen main links service_status [list launch_browser "http://messenger.msn.com/Status.aspx"]
 		$self CanvasTextToLink login_screen main links new_account [list launch_browser "https://accountservices.passport.net/reg.srf?sl=1&lc=1033"]
-		$self CanvasTextToLink login_screen main check_ver text "::autoupdate::check_version"
 
 		$self FillLoginFields
 
@@ -455,7 +445,6 @@
  		contentmanager show login_screen main checkboxes
  		contentmanager show login_screen main login
  		contentmanager show login_screen main links
- 		contentmanager show login_screen main check_ver
  		contentmanager show login_screen main more
 
 		# We sort to get the sizes of each element..
@@ -470,7 +459,6 @@
  		contentmanager hide login_screen main checkboxes
  		contentmanager hide login_screen main login
  		contentmanager hide login_screen main links
- 		contentmanager hide login_screen main check_ver
  		contentmanager show login_screen main more
 
  		set max [winfo height $self]
@@ -512,12 +500,8 @@
 					if { $current < $max } {
 						contentmanager show login_screen lang
 
-						incr current [contentmanager height login_screen main check_ver]
-						incr current [contentmanager cget login_screen main check_ver -pady]
-						incr current [contentmanager cget login_screen main check_ver -pady]
 						
 						if { $current < $max } {
-							contentmanager show login_screen main check_ver
 
 							if { [::config::getKey show_login_screen_links 1] } {
 								incr current [contentmanager height login_screen main links]
@@ -565,7 +549,6 @@
  		contentmanager show login_screen main checkboxes
  		contentmanager show login_screen main login
  		contentmanager show login_screen main links
- 		contentmanager show login_screen main check_ver
  		contentmanager show login_screen main more
 
 		# We sort to get the sizes of each element..
